akka-http-tests/src/test/scala/akka/http/scaladsl/server/EntityStreamingSpec.scala

```diff
@@ -278,7 +279,8 @@ class EntityStreamingSpec extends RoutingSpec with ScalaFutures {
               .runFold(0) { (cnt, _) => cnt + 1 }
 
           complete {
-            measurementsSubmitted.map(n => Map("msg" -> s"""Total metrics received: $n"""))
+            implicit val preciseFormat: RootJsonFormat[Map[String, String]] = mapFormat // workaround for Scala 3
+            ToResponseMarshallable(measurementsSubmitted.map((n: Int) => Map[String, String]("msg" -> s"""Total metrics received: $n""")))
           }
```

Comment:
This part is interesting I think - without the additional implicit brought closer in scope the compiler would claim to have found conflicting RootJsonFormats in `CollectionConverters` -  they were `iterableFormat` and `immIterableFormat` instead of mapFormat. Not sure why that is yet.